exporter/prometheusremotewriteexporter/factory.go

```diff
@@ -66,6 +66,11 @@ func createMetricsExporter(_ context.Context, params component.ExporterCreatePar
 		params.Logger,
 		prwe.PushMetrics,
 		exporterhelper.WithTimeout(prwCfg.TimeoutSettings),
+		exporterhelper.WithQueue(exporterhelper.QueueSettings{
+			Enabled:      true,
+			NumConsumers: 1,
+			// TODO(jbd): Allow users to modify the queue size.
```

Comment:
Probably we should set a default size

Reply:
Added a default, we can adjust this based on the benchmarking work we're planning to do soon. The default is not a very informed guess at this point.